lib/index.js

```diff
@@ -402,20 +402,28 @@ client.on('message', message => {
         // document command in help section
       case 'dotprefix':
         if (isAdmin) {
-          db.updateServerPrefix(
-            serverId, args[0]
-          ).then(data => {
-            serversConfigStore = []
-            if (data) client.guilds.cache.forEach(server => {
-              db.saveServerConfig(
-                +new Date, server.id //rework schema to eliminate the need of pushing date unecessarily here or make a find function
-              ).then(data => {
-                serversConfigStore.push(data)
+          if(args[0] != undefined) {
+            if(args[0].length == 1) {
+             db.updateServerPrefix(
+               serverId, args[0]
+             ).then(data => {
+               serversConfigStore = []
+               if (data) { message.channel.send("Prefix changed."); } if (data) client.guilds.cache.forEach(server => {
+                db.saveServerConfig(
+                   +new Date, server.id //rework schema to eliminate the need of pushing date unecessarily here or make a find function
+                ).then(data => {
+                   serversConfigStore.push(data)
+                })
               })
-            })
-            serversConfig = serversConfigStore;
-          })
+              serversConfig = serversConfigStore;
+             })
+           } else {
+             message.channel.send("Max length is 1."); 
+           }
+        } else {
+        	message.channel.send("Min length is 1."); 
         }
```

Comment:
```suggestion
        }
```
This will be an uneccessary else statement due to removal of if statement in the beginning.